wait_test.go

```diff
@@ -58,10 +58,14 @@ func Test_Wait(t *testing.T) {
 
 		urls := []string{
 			"https://" + newTestAddr(t) + "/foo/bar",
-			newTestAddr(t) + "/foo/bar",
 			"https://" + newTestAddr(t),
+			"http://" + newTestAddr(t) + "/foo/bar",
+			"http://" + newTestAddr(t),
+			newTestAddr(t),
 		}
+
 		for _, url := range urls {
+			t.Log(url)
```

Comment:
q: Do we want this log here?

Reply:
I added it because this broke and I wasn't sure which URL it was, but probably not the best way to handle it if `t.Log` is going to pollute our output on verbose tests.

Went with named sub-tests, could also swap to adding the info to the `assert` statement if that makes more sense.